src/containers/profile.js

```diff
@@ -0,0 +1,127 @@
+        <div className="col-md-offset-3 col-md-6 bg-white content">
+          <div className="col-md-12">
+            <div className="text-align-center">
+              <h3>User profile</h3>
+            </div>
+          </div>
+          <div className="col-md-12 mt-20px">
+            <div className="col-md-3 pull-right">
+              <Button handleSubmitProps={this.enableFields} btnValue="Edit" />
+            </div>
+            <div className="col-md-9">
+              <div className="col-md-12">
+                <div className="col-md-3">
+                  <label>Name:</label>
+                </div>
+                <div className="col-md-6">
+                  <input
+                    type="text"
+                    value={name}
+                    disabled={this.state.fieldDisable}
+                  />
+                </div>
+              </div>
+              <div className="col-md-12 mt-20px">
+                <div className="col-md-3">
+                  <label>Age:</label>
+                </div>
+                <div className="col-md-6">
+                  <input
+                    type="text"
+                    value={Age}
+                    disabled={this.state.fieldDisable}
+                  />
+                </div>
+              </div>
+              <div className="col-md-12 mt-20px">
+                <div className="col-md-3">
+                  <label>Contact:</label>
+                </div>
+                <div className="col-md-6">
+                  <input
+                    type="text"
+                    value={Contact}
+                    disabled={this.state.fieldDisable}
+                  />
+                </div>
+              </div>
+              <div className="col-md-12 mt-20px">
+                <div className="col-md-3">
+                  <label>Address:</label>
+                </div>
+                <div className="col-md-6">
+                  <div>
+                    <input
+                      type="text"
+                      value={Address1}
+                      disabled={this.state.fieldDisable}
+                    />
+                  </div>
+                  <div className="mt-20px">
+                    <input
+                      type="text"
+                      placeholder="Address field"
+                      value={Address2}
+                      disabled={this.state.fieldDisable}
+                    />
+                  </div>
+                  <div className="mt-20px">
+                    <input
+                      type="text"
+                      placeholder="Address field"
+                      value={Address3}
+                      disabled={this.state.fieldDisable}
+                    />
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div className="col-md-offset-5 col-md-2 mt-20px">
+              {this.state.fieldDisable ? null : <Button btnValue="Update" />}
```

Comment:
no need to make use of state, when u have state management library.
Define it in the redux state, and the same thing will be available in the props